docs/02-app/01-building-your-application/06-optimizing/13-memory-usage.mdx

```diff
@@ -13,6 +13,12 @@ Applications with a large amount of dependencies will use more memory.
 
 The [Bundle Analyzer](/docs/app/building-your-application/optimizing/bundle-analyzer) can help you investigate large dependencies in your application that may be able to be removed to improve performance and memory usage.
 
+## Run `next build` with `--experimental-debug-memory-usage`
+
+Starting in `14.2.0`, you can run `next build --experimental-debug-memory-usage` to run the build in a mode where Next.js
+will print out information about memory usage continuously throughout the build, such as heap usage and garbage collection
+statistics. Heap snapshots will also be taken automatically when memory usage gets close to the configured limit.
```

Comment:
Should we add a note about the `webpackBuildWorker` caveat?

Reply:
Added.